app/src/main/java/edu/uc/coffeens/stocktracker/dao/IStockDAO.kt

```diff
@@ -13,7 +12,4 @@ interface IStockDAO {
     @GET("/stevekinney/f96d5800852e91282f46/raw/ea047c2f5898de6c9fecf535db8b30abcfe5b423/stocks.json")
     fun getAllStocks(): Call<ArrayList<Stock>>
 
-    @GET("/stevekinney/f96d5800852e91282f46/raw/ea047c2f5898de6c9fecf535db8b30abcfe5b423/stocks.json")
-    fun getStock(@Query("Combined_Name") name: String): Call<ArrayList<Stock>>
-
```

Comment:
Queried GET was unused and presumed unneeded so I removed along with the import

Reply:
This duplicates a previous code review.  When that occurs, first in wins.